### DISCUSSION TITLE
Test interrupteur zigbee ZTS-EU à bouton poussoir Moes - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#36] [Comments]
# Test interrupteur zigbee ZTS-EU à bouton poussoir Moes - Haade.fr

Après le test de l’interrupteur tactile Moes, pour des raisons personnelles j’ai commandé le même à bouton poussoir

https://haade.fr/fr/blog/test-bouton-poussoir-moes-zts-eu-zigbee

**Comment by SephiWaffle** [comment]
Bonjour et merci pour ce test, c'est toujours très utile avant achat.
Il est mentionné que si ces interrupteurs sont en théorie routeurs (lorsque connectés au neutre), ils étaient vus comme simples end device à l'époque. Vous sauriez si c'est réglé svp ?
Merci

**Reply by haade-administrator** [reply]
Bonjour, non les modules qui n'ont pas besoin du neutre pour fonctionner ne posséderont pas cette capacité à fonctionner comme routeur, car ils ne sont pas alimentés en permanence malheureusement même en branchant le neutre qui est un leurre
